tests/service/FsUnit.fs

```diff
@@ -1,38 +1,6 @@
 ﻿module FsUnit
 
-open System.Diagnostics
 open Xunit
 
-[<DebuggerNonUserCode>]
-let should (f: 'a -> objnull -> unit) x (y: obj) =
-    let y =
-        match y with
-        | :? (unit -> unit) -> box (fun () -> (y :?> unit -> unit))
-        | _                 -> y
-    f x y
-
-/// Note, xunit does check types by default.
-/// These are artifacts of nunit and not necessary now, just used in many places.
-let equal (expected: 'a) (actual: 'a) = 
-    Assert.Equal<'a>(expected, actual)
-
 let shouldEqual (x: 'a) (y: 'a) =
-    Assert.Equal<'a>(x, y)
-
-/// Same as 'shouldEqual' but goes pairwise over the collections. Lengths must be equal.
-let shouldPairwiseEqual (x: seq<_>) (y: seq<_>) =
```

Comment:
xUnit is comparing collection pairwise by default, this shouldn't be need at all.